src/main/java/net/fabricmc/installer/Handler.java

```diff
@@ -141,39 +144,43 @@ private void appendException(StringBuilder errorMessage, String prefix, String n
 		}
 
 		if (e.getCause() != null) {
-			appendException(errorMessage, prefix + prefixAppend, "Caused by", e.getCause());
+			appendException(errorMessage, prefix + prefixAppend, translate("prompt.exception.caused.by"), e.getCause());
 		}
 
 		for (Throwable ec : e.getSuppressed()) {
-			appendException(errorMessage, prefix + prefixAppend, "Suppressed", ec);
+			appendException(errorMessage, prefix + prefixAppend, translate("prompt.exception.suppressed"), ec);
 		}
 	}
 
 	@Override
 	public void error(Exception e) {
 		StringBuilder errorMessage = new StringBuilder();
-		appendException(errorMessage, "", "Exception", e);
+		appendException(errorMessage, "", translate("prompt.exception"), e);
 
 		System.err.println(errorMessage);
 
 		JOptionPane.showMessageDialog(
 				pane,
 				errorMessage,
-				"Exception occured!",
+				translate("prompt.exception.occurrence"),
 				JOptionPane.ERROR_MESSAGE
 		);
 
 		statusLabel.setText(e.getLocalizedMessage());
 		statusLabel.setForeground(Color.RED);
 	}
 
+	private String translate(String key) {
```

Comment:
Some stuff uses this, but not all things. Either get rid of this method or make everything use it.

Reply:
Everything now uses the static final field directly.